CODEOWNERS

```diff
@@ -204,7 +204,7 @@ k8s/**/common/bsp/identity.yaml @hmcts/bulk-scan
 k8s/**/common/bsp/bulk-scan-payment-processor.yaml @hmcts/bulk-scan
 k8s/**/common/bsp/bulk-scan-processor.yaml @hmcts/bulk-scan
 k8s/**/common/bsp/bulk-scan-orchestrator.yaml @hmcts/bulk-scan
-k8s/aat/common/bsp/bulk-scan-sample-app.yaml @hmcts/bulk-scan
```

Comment:
this is giving u access to deploy this to prod. so, pls revert.

Reply:
sample should not go to prod at all. but yea, perhaps worth just enlisting to be sure. unless there is some sort of _ignore_ list to override this setting